customScripts/expansionsetup.gd

```diff
@@ -481,8 +481,18 @@ func setLactation(person):
 						person.add_trait("Milk Glands 1")
 				traitchance -= 1
 
-###---Added by Expansion---### Deviate Concept Tweaked
+#ralph4
+func cleargenes(person):
+	for race in person.genealogy:
+		person.genealogy[race] = 0
+#/ralph4
+
+###---Added by Expansion---### Deviate Concept Tweaked ###Ralphomod### tweaked further
 func setRaceBonus(person, increasestats):
+	var count_races = 0
```

Comment:
There are significant changes to existing races without condition, due to the vast differences it may be best to put this in a new function. Something like
func setRaceBonus(person, increasestats):
    if useRalphsTweaks:
        setRaceBonus_Ralphs(person, increasestats)
        return

Reply:
I'll work on this tomorrow lol

Reply:
Thank you so much for doing this one for me.